src/sentry/conf/server.py

```diff
@@ -2711,7 +2711,7 @@ def build_cdc_postgres_init_db_volume(settings):
 
 SNOWFLAKE_VERSION_ID = 1
 SNOWFLAKE_REGION_ID = 0
-
+SENTRY_SNOWFLAKE_EPOCH_START = datetime(2022, 7, 21, 0, 0).timestamp()
```

Comment:
@maxiuyuan we can probably bump this up to August now :D

Reply:
will do!